docs/src/app/(public)/(content)/react/utils/use-render/demos/render/css-modules/index.tsx

```diff
@@ -0,0 +1,29 @@
+'use client';
+import * as React from 'react';
+import { useRender } from '@base-ui-components/react/use-render';
+import styles from './index.module.css';
+
+type TextProps = {
+  render?: useRender.RenderProp<Record<string, unknown>>;
```

Comment:
`Record<string, unknown>` is the default. Omitting it makes the code cleaner.

```suggestion
  render?: useRender.RenderProp;
```

Reply:
The default is `Record<string, any>`, should it be updated to `Record<string, unknown>`?

https://github.com/mui/base-ui/pull/1418/files#diff-1ed79dc7239102d3f0a279612e8c0e98471ef4821c8427d59d47c46d16b90b13R24-R33

Reply:
Yep, changed.